labgrid/autoinstall/main.py

```diff
@@ -0,0 +1,203 @@
+"""The autoinstall.main module runs an installation script automatically on multiple targets."""
+import ast
+import argparse
+import logging
+import sys
+import traceback
```

Comment:
Unnecessary import, not used in this file, same for:
- `attr`
- `from contextlib import closing`